include/mbedtls/ccm.h

```diff
@@ -308,6 +308,27 @@ int mbedtls_ccm_star_auth_decrypt( mbedtls_ccm_context *ctx, size_t length,
  * \param iv_len    The length of the nonce in Bytes: 7, 8, 9, 10, 11, 12,
  *                  or 13. The length L of the message length field is
  *                  15 - \p iv_len.
+ *
```

Comment:
In the documentation of starts(), please state that set_lengths() needs to be called before passing any data, and that is can be called before or after starts().

In the documentation of set_lengths(), please state that this function may be called before or after starts(), and must be called before passing any data.